### ISSUE TITLE
Accessibility: Content not included in landmarks - Mobile Logo

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#438]
**Description**
All content on a webpage should be inside landmarks. This allows screen readers to understand the purpose of different areas of the page at a glance based on the defined roles that these landmarks have.

**Solution**
The code for the Cornell Mobile Logo needs to be moved within the `<header>` tag.

**Comment by mhk33** [comment]
Fixed in dev/3.0 upgrade.
